ote/src/cljs/ote/app/controller/front_page.cljs

```diff
@@ -67,11 +77,8 @@
 
   EnsureTransportOperator
   (process-event [_ app]
-    (if (empty? (get app :transport-operator))
-      (do
-        (routes/navigate! :no-operator)
-        (assoc app :page :no-operator)
-        )
+     (if (get app :services-changed?)
```

Comment:
It is clearer to use `(:services-changed? app)` instead of using `get` function. Or just destructure the wanted key in the parameter declaration,